Xamarin.Forms.Platform.UAP/Resources.xaml

```diff
@@ -5,17 +5,18 @@
 	x:Class="Xamarin.Forms.Platform.UWP.Resources">
 
 	<ResourceDictionary.MergedDictionaries>
-        <ResourceDictionary Source="FormsCommandBarStyle.xaml" />
+		<ResourceDictionary Source="FormsCommandBarStyle.xaml" />
 		<ResourceDictionary Source="PageControlStyle.xaml" />
 		<ResourceDictionary Source="FormsProgressBarStyle.xaml" />
 		<ResourceDictionary Source="FormsTextBoxStyle.xaml" />
+		<ResourceDictionary Source="FormsCheckBoxStyle.xaml" />
```

Comment:
This is the only actual change to this file. This file has a mix of tabs and spaces which makes editing just parts of it without the IDE doing things fairly annoying. I can roll back all the formatting if we really want but it'd be nice to just have this file be consistent